projects/plugins/jetpack/modules/search/instant-search/loader.js

```diff
@@ -0,0 +1,28 @@
+// NOTE: This must be imported first before any other imports.
+// See: https://github.com/webpack/webpack/issues/2776#issuecomment-233208623
+import './set-webpack-public-path';
+
+/**
+ * Internal dependencies
+ */
+import { SERVER_OBJECT_NAME } from './lib/constants';
+import { bindCustomizerChanges } from './lib/customize';
+
+/**
+ * Loads and runs the main chunk for Instant Search.
+ */
+function init() {
+	import( './index' ).then( instantSearch => instantSearch.initialize() );
```

Comment:
Could we give this chunk a static name so that we can measure its size [via size-limit](https://github.com/Automattic/jetpack/blob/update/lazy-load-instant-search/projects/plugins/jetpack/.size-limit.js)? 

Also, it might be nice to set a size limit for the newly-slimmed-down loader bundle; what do you think would be a reasonable upper limit for the new critical path bundle?

Reply:
Done and done! 🙂

I went with 4KB for the new critical path bundle, to give it just a little bit of room to grow, if need be. There shouldn't be any need for it, to be honest, but I think it's ok to leave it at that.

I kept the lazily loaded chunk limit at 50KB.